staging/gatekeeper/README.md

```diff
@@ -34,25 +34,23 @@ Gatekeeper.
-| securityContext.runAsUser                             | Run as user for gatekeeper pod                                               |                     |
-| securityContext.fsGroup                               | FS group for gatekeeper pod                                                  |                     |
-| service.annotations                                   | Service annotations                                                          |                     |
-| service.type                                          | Service type                                                                 | "ClusterIP"         |
-| service.port                                          | Service port                                                                 | 443                 |
-| rbac.create                                           | Create RBAC resources                                                        | true                |
-| serviceAccount.create                                 | Create service account                                                       | true                |
-| serviceAccount.name                                   | Service account name (if not specified, will be generated from release name) |                     |
-| priorityClassName                                     | Pod priority class name                                                      |                     |
-| resources                                             | Pod resources                                                                |                     |
-| admissionControllerFailurePolicy                      | Admission controller failure policy                                          | "Ignore"            |
-| admissionControllerNamespaceSelector.matchExpressions | Admission controller namespace selector expressions                          | []                  |
-| admissionControllerObjectSelector.matchExpressions    | Admission controller object selector expressions                             | []                  |
-| admissionControllerObjectSelector.matchLabels         | Admission controller object label selector                                   | []                  |
-| webhook.apiGroups                                     | Webhook rules API groups                                                     | [""]                |
-| webhook.apiVersions                                   | Webhook rules API versions                                                   | ["*"]               |
-| webhook.operations                                    | Webhook rules operations                                                     | ["CREATE","UPDATE"] |
-| webhook.resources                                     | Webhook rules resources                                                      | ["pods"]            |
```

Comment:
are we removing these properties on purpose ?

Reply:
Yes, changed to be more flexible by allowing multiple rules to be defined.